specification/machinelearningservices/resource-manager/readme.md

```diff
@@ -27,7 +27,7 @@ These are the global settings for the Machine Learning Services API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2021-07-01
+tag: package-2021-03-01-preview
```

Comment:
We should never downgrade the default tag - this will impact all the SDKs
The SDK can individually set the tag they want to use, I assume